src/lib/base-command.js

```diff
@@ -118,13 +117,13 @@ class BaseCommand extends Command {
   }
 
   doLog(message, always = false) {
-    if (always || !this.flags.quiet) {
-      CliUx.ux.log(message);
+    if (always || !this.flags?.quiet) {
+      this.log(message);
     }
   }
```

Comment:
I wonder if it wouldn't be nicer to override `Command#log` with the quiet flag check, delegating back to the super class when desired. See https://github.com/oclif/core/blob/main/src/command.ts#L257-L262

Reply:
Would be nicer yes, but when we move to TS ;-)